WebDriverAgentMac/WebDriverAgentLib/PrivateHeaders/XCTest/XCPointerEvent.h

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
Which Xcode version base? (Just a question)

Reply:
ah, or just copy from iOS-WDA?

Reply:
XCode12

headers for macOS and iOS are slightly different